robotiq_hande_driver/include/communication.hpp

```diff
@@ -1,43 +1,57 @@
 #ifndef COMMUNICATION_HPP_
 #define COMMUNICATION_HPP_
 
-#include <stdint.h>
-#include <stddef.h>
-#include <vector>
 #include <modbus/modbus.h>
+#include <string.h>
```

Comment:
Use the C++ standard library instead of the C library:
```suggestion
#include <string>
```

Reply:
using `cstring`